var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -364,6 +366,11 @@ def configure_args(self):
             else:
                 config_args.append('--without-cuda')
 
+        # UCX support
+        if spec.satisfies('@3.0:'):
```

Comment:
Please add a conflict for older versions

Reply:
That should have been there....